.dscanner.ini

```diff
@@ -458,19 +458,18 @@ unused_variable_check="-std.algorithm.comparison,\
 -std.range,\
 -std.range.interfaces,\
 -std.range.primitives,\
--std.regex.internal.backtracking,\
--std.regex.internal.tests,\
--std.regex.internal.thompson,\
+-std.regex.internal.,\
```

Comment:
Needs `std.path` here until https://github.com/dlang-community/D-Scanner/issues/727 is fixed.

due to:

```
std/path.d(2659:26)[warn]: Parameter path is never used.
```